### ISSUE TITLE
An error occurred while reading the F# metadata node at position 1 in table 'itycons' of assembly 'WebSharper.Core.JavaScript, Version=4.5.0.0

### ISSUE THREAD

**Issue by amieres** [dotnet-websharper/core#1029]
The title message shows when building the Suave Hosted-Site project using WebSharper  4.5.2.275. 

Same error also shows when trying to compile using wsfsc.exe with versions 4.5.2.290 & 4.5.3.310 from Nuget.

```
1>------ Rebuild All started: Project: Suave1, Configuration: Debug Any CPU ------
1>D:\Programs\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin\Microsoft.Common.CurrentVersion.targets(2110,5): warning MSB3277: Found conflicts between different versions of "FSharp.Core" that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed.
1>		Consider app.config remapping of assembly "Microsoft.Owin, Culture=neutral, PublicKeyToken=31bf3856ad364e35" from Version "3.0.0.0" [] to Version "4.0.0.0" [C:\Users\Abelardo\Source\Repos\Suave1\packages\Microsoft.Owin.4.0.0\lib\net451\Microsoft.Owin.dll] to solve conflict and get rid of warning.
1>D:\Programs\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin\Microsoft.Common.CurrentVersion.targets(2110,5): warning MSB3276: Found conflicts between different versions of the same dependent assembly. Please set the "AutoGenerateBindingRedirects" property to true in the project file. For more information, see http://go.microsoft.com/fwlink/?LinkId=294190.
1>FSC: warning FS3186: An error occurred while reading the F# metadata node at position 1 in table 'itycons' of assembly 'WebSharper.Core.JavaScript, Version=4.5.0.0, Culture=neutral, PublicKeyToken=dcd983dec8f76a71'. The node had no matching declaration. Please report this warning. You may need to recompile the F# assembly you are using.
1>FSC: warning FS3186: An error occurred while reading the F# metadata node at position 0 in table 'ivals' of assembly 'WebSharper.Core.JavaScript, Version=4.5.0.0, Culture=neutral, PublicKeyToken=dcd983dec8f76a71'. The node had no matching declaration. Please report this warning. You may need to recompile the F# assembly you are using.
1>		Suave1 -> C:\Users\Abelardo\Source\Repos\Suave1\bin\Debug\Suave1.exe
========== Rebuild All: 1 succeeded, 0 failed, 0 skipped ==========
```

When it runs it throws exception with the following messages:
```
'Suave1.exe' (CLR v4.0.30319: DefaultDomain): Loaded 'C:\WINDOWS\Microsoft.Net\assembly\GAC_64\mscorlib\v4.0_4.0.0.0__b77a5c561934e089\mscorlib.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled.
'Suave1.exe' (CLR v4.0.30319: DefaultDomain): Loaded 'C:\Users\Abelardo\Source\Repos\Suave1\bin\Debug\Suave1.exe'. Symbols loaded.
'Suave1.exe' (CLR v4.0.30319: Suave1.exe): Loaded 'C:\Users\Abelardo\Source\Repos\Suave1\bin\Debug\WebSharper.Sitelets.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled.
'Suave1.exe' (CLR v4.0.30319: Suave1.exe): Loaded 'C:\Users\Abelardo\Source\Repos\Suave1\bin\Debug\FSharp.Core.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled.
'Suave1.exe' (CLR v4.0.30319: Suave1.exe): Loaded 'C:\Users\Abelardo\Source\Repos\Suave1\bin\Debug\Suave.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled.
'Suave1.exe' (CLR v4.0.30319: Suave1.exe): Loaded 'C:\Users\Abelardo\Source\Repos\Suave1\bin\Debug\WebSharper.Web.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled.
'Suave1.exe' (CLR v4.0.30319: Suave1.exe): Loaded 'C:\Users\Abelardo\Source\Repos\Suave1\bin\Debug\WebSharper.Core.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled.
'Suave1.exe' (CLR v4.0.30319: Suave1.exe): Loaded 'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\System\v4.0_4.0.0.0__b77a5c561934e089\System.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled.
'Suave1.exe' (CLR v4.0.30319: Suave1.exe): Loaded 'C:\Users\Abelardo\Source\Repos\Suave1\bin\Debug\WebSharper.Main.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled.
An unhandled exception of type 'System.IO.FileLoadException' occurred in Unknown Module.
Could not load file or assembly 'FSharp.Core, Version=4.4.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)

The program '[7768] Suave1.exe' has exited with code -1 (0xffffffff).

```

**Comment by Jand42** [comment]
Hi, the F# metadata warning is not causing the problem with the Suave template, it is a missing redirection needed by newer Suave. Added a separate ticket for that: https://github.com/dotnet-websharper/templates/issues/20

**Comment by Tarmil** [comment]
Yeah these warnings started appearing with the FCS upgrade that came with WS 4.5.3, we're still figuring them out. They're benign though so you can safely ignore them with `<NoWarn>3186</NoWarn>` in your fsproj (we've fixed wsfsc to take NoWarn into consideration correctly like fsc does).

**Comment by Jand42** [comment]
Reverting FCT version used to built WebSharper.Core.JavaScript.dll seems to solve this problem. This is now being published with WebSharper 4.5.8.327.

**Comment by Jand42** [comment]
This is an issue again, since using latest build tools again. Adding a `nowarn` for it implicitly. (It is benign, see https://github.com/dotnet/fsharp/issues/592)
